main.py

```diff
@@ -96,26 +97,34 @@ def userSelectedCharacter(self, character: Character):
 
     def displayCharacterDetails(self, character):
         pygame.mixer.music.stop()
-        self.TypeWriterSound = pygame.mixer.Sound(AUDIO["TypeWriterSound"])
-        self.TypeWriterSound.set_volume(0.9)
-        self.TypeWriterSound.play(loops=-1)
         # Check if the character's name is "John" for special handling
         if character.name == "John":
+            self.TypeWriterSound = pygame.mixer.Sound(AUDIO["TypeWriterSound"])
+            self.TypeWriterSound.set_volume(0.9)
+            self.TypeWriterSound.play(loops=-1)
             # Load John's theme music
             pygame.mixer.music.load(AUDIO["JohnTheme"])
             # Set the volume to maximum (1.0)
             pygame.mixer.music.set_volume(0.4)
             # Play John's theme music in a loop indefinitely
             pygame.mixer.music.play(-1)
 
-        self.characterWidget = CharacterDetailWidget(
-            character, transition_callback=self.showMapScreen
-        )
+            self.characterWidget = CharacterDetailWidget(
+                character, transition_callback=self.showMapScreen
+            )
+        elif character.name == "Cooper":
+            self.characterWidget = CooperIntroScreen(character, transition_callback=self.showMapScreen)
+        else:
+            raise Exception(f'Introscreen for {character.name} has not yet been implemented')
```

Comment:
@NoahBartell @MLHumphries make sure to make your own and set it to `self.characterWidget`